data/human/ships.txt

```diff
@@ -2572,6 +2572,42 @@ ship "Manta"
 
 
 
+ship "Mining Drone"
+	sprite "ship/mining drone"
+	thumbnail "thumbnail/mining drone"
+	attributes
+		category "Drone"
+		"cost" 83000
```

Comment:
Same exact base cost as a combat drone? And with more expensive outfits, that makes the mining drone more expensive to buy than the combat drone.
```suggestion
		"cost" 58000
```